languages/python/reference/exercise-concepts/reverse-string.md

```diff
@@ -0,0 +1,41 @@
+## Concept extraction for reverse-string
+
+Example implementation:
+```python
+def reverse(text: str) -> str:
+    """Reverse a given string
```

Comment:
I think we can dispense with the docstring, it doesn't add anything but noise to the solution.

Reply:
Thank you for your suggestion.